sovtokenfees/sovtokenfees/test/auth_map/test_auth_rules.py

```diff
@@ -0,0 +1,521 @@
+    return helpers.general.do_mint(outputs)
+
+
+@pytest.fixture(scope='module')
+def sdk_wallet_stewards(looper,
+                        sdk_wallet_trustee,
+                        sdk_pool_handle):
+    sdk_wallet_stewards = []
+    for i in range(3):
+        wallet = sdk_add_new_nym(looper,
+                                 sdk_pool_handle,
+                                 sdk_wallet_trustee,
+                                 alias='steward{}'.format(i),
+                                 role=STEWARD_STRING)
+        sdk_wallet_stewards.append(wallet)
+    return sdk_wallet_stewards
+
+
+@pytest.fixture(scope='module')
+def sdk_wallet_trust_anchors(looper,
+                        sdk_wallet_trustee,
+                        sdk_pool_handle):
+    sdk_wallet_stewards = []
+    for i in range(3):
+        wallet = sdk_add_new_nym(looper,
+                                 sdk_pool_handle,
+                                 sdk_wallet_trustee,
+                                 alias='trust_anchors{}'.format(i),
+                                 role=TRUST_ANCHOR_STRING)
+        sdk_wallet_stewards.append(wallet)
+    return sdk_wallet_stewards
+
+
+@pytest.fixture(scope='module')
+def sdk_wallet_clients(looper,
+                       sdk_wallet_trustee,
+                       sdk_pool_handle):
+    sdk_wallet_clients = []
+    for i in range(3):
+        wallet = sdk_add_new_nym(looper,
+                                 sdk_pool_handle,
+                                 sdk_wallet_trustee,
+                                 alias='client{}'.format(i))
+        sdk_wallet_clients.append(wallet)
+    return sdk_wallet_clients
+
+
+@pytest.fixture(scope='module')
+def sdk_wallet_trustees(looper,
+                        sdk_wallet_trustee,
+                        sdk_pool_handle):
+    sdk_wallet_trustees = []
+    for i in range(3):
+        wallet = sdk_add_new_nym(looper,
+                                 sdk_pool_handle,
+                                 sdk_wallet_trustee,
+                                 alias='trustee{}'.format(i),
+                                 role=TRUSTEE_STRING)
+        sdk_wallet_trustees.append(wallet)
+    return sdk_wallet_trustees
+
+
+@pytest.fixture()
+def address(helpers):
+    return helpers.wallet.create_address()
+
+
+def add_fees_request_with_address(helpers, fee_amount, request, address):
+    utxos_found = helpers.general.get_utxo_addresses([address])[0]
+    request_with_fees = helpers.request.add_fees(
+        request,
+        utxos_found,
+        fee_amount,
+        change_address=address
+    )[0]
+    request_with_fees = json.loads(request_with_fees)
+    setattr(request, FEES, request_with_fees[FEES])
+    return request
+
+
+def _send_request(looper, helpers, fees, wallets_count, address, owner, sdk_wallet_trustee,
+                  sdk_wallet_trustees, sdk_wallet_stewards,
+                  sdk_wallet_clients, sdk_wallet_trust_anchors):
+    print(wallets_count)
```

Comment:
Do we need this print?

Reply:
Yes, I think. Because it's only one way to understand which request failed. We can change print to writing in logs.